src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -565,7 +565,7 @@ private static size_t CurlReceiveHeadersCallback(IntPtr buffer, size_t size, siz
                             if (colonIndex > 0)
                             {
                                 string headerName = responseHeader.Substring(0, colonIndex);
-                                string headerValue = responseHeader.Substring(colonIndex + 1);
+                                string headerValue = responseHeader.Substring(colonIndex + 1).Trim();
```

Comment:
Nit:
Trim will remove characters which return true for ```Char.IsWhitespace```. Right? 
The [spec](http://tools.ietf.org/html/rfc7230#section-1.2) says we can have what it calls ```OWS``` here (optional whitespace). And OWS is defined as SPACE | HTAB.

Would it be possible to use ```Trim(' ', '\t')``` here? Theoretically, CurlHandler could end up having different values for headers. 
Or, if needed I can handle it in #4131